README.md

```diff
@@ -1,10 +1,32 @@
 # automate
-Created with Electron JS and integrating with both Azure DevOps and GitHub, automate allows you to automate tasks such as changing branches across enterprise workspaces and solutions, tagging multiple repositories at the same time, and much more.
+automate is an Electron JS based source management program with Azure DevOps and Github integration. automate focuses on streamlining workflow tasks such as changing branches across enterprise workspaces and solutions, as well as tagging multiple repositories at the same time.
+
 
  [![Screenshot of the landing page.](images/landing.png)]()
 
-## Getting Started
-Make sure that you have [Node.js](https://nodejs.org/en/) installed. The minimum required version is 14.
+## Features
+* **Branch management:** automate branch management across enterprise workspaces and solutions to organize your workflow 
+* **Tagging multiple repositories:** tag multiple repositories simultaneously to optimize your performance
+* **User-oriented:** planned real-time theme customization for a user experience that fits your personal needs
+
+
+## Installation
+1. Install [Node.js](http://nodejs.org/en/) - the recommended version is 14.0.0 or above
+2. Download automate files and extract them to a new folder
+3. Open command prompt and navigate to the root directory for automate
+4. Run `npm install` to install local dependencies
+5. Run `npm start` to launch the program
+
+
+## Contribute
+User feedback for automate is welcomed and encouraged. Every contribution helps to develop the project further
+
+There are many ways for you to contribute to the project: 
+
+* Log [bugs and report issues](https://github.com/tacosontitan/automate/issues), and help verify reports
+* Verify [changes to source code](https://github.com/tacosontitan/automate/pulls)
+* Review the [documentation](https://github.com/tacosontitan/automate/pulls) and submit pull requests for any edits ranging from typos to new content
+
 
 ## Configuration
```

Comment:
Again, I know little about coding so I have no clue how essential this is to the setup process for the user, and could not figure out the configuration on my own. If this is information directed at the developers/any contributing developers, I think a full documentation tab somewhere within the repository with a code index might be a better location for this information